app/src/main/java/com/m2049r/xmrwallet/util/Helper.java

```diff
@@ -508,6 +508,12 @@ public void onAuthenticationError(int errMsgId, CharSequence errString) {
                     tvOpenPrompt.setText(errString);
                 }
 
+                @Override
+                public void onAuthenticationHelp(int helpMsgId, CharSequence helpString) {
```

Comment:
can this be problem since we are supporting down to API 21 and this was added in API 23?

Reply:
If the device doesn't support API 23, fingerprint-related function will be disabled and not be executed.
Tested on Android Emulator (API 21, x86).